src/DssSpell.sol

```diff
@@ -67,264 +51,55 @@ contract DssSpellAction is DssAction {
-        VestLike(MCD_VEST_DAI).restrict(
-            VestLike(MCD_VEST_DAI).create(
-                JANSKY,                    // usr
-                504_000 * WAD,             // tot
-                OCT_01_2023,               // bgn
-                SEP_30_2024 - OCT_01_2023, // tau
-                0,                         // eta
-                address(0)                 // mgr
-            )
-        );
-        // VoteWizard | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 504,000.00 DAI | 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
-        VestLike(MCD_VEST_DAI).restrict(
-            VestLike(MCD_VEST_DAI).create(
-                VOTEWIZARD,                // usr
-                504_000 * WAD,             // tot
-                OCT_01_2023,               // bgn
-                SEP_30_2024 - OCT_01_2023, // tau
-                0,                         // eta
-                address(0)                 // mgr
-            )
-        );
 
-        // Increase allowance by new vesting delta
-        uint256 newVesting = 216 ether; // JANSKY; note: ether is a keyword helper, only MKR is transferred here
-               newVesting += 216 ether; // VOTEWIZARD; note: ether is a keyword helper, only MKR is transferred here
-        MKR.approve(address(MCD_VEST_MKR_TREASURY), MKR.allowance(address(this), (address(MCD_VEST_MKR_TREASURY))) + newVesting);
+        // ---------- Spark - AAVE Revenue Share Payment ----------
+        // Forum: https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-1-q3-2023/22486
+        // MIP: https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share
 
-        // JanSky | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 216.00 MKR | 0xf3F868534FAD48EF5a228Fe78669cf242745a755
-        VestLike(MCD_VEST_MKR_TREASURY).restrict(
-            VestLike(MCD_VEST_MKR_TREASURY).create(
-                JANSKY,                    // usr
-                216 ether,                 // tot
-                OCT_01_2023,               // bgn
-                SEP_30_2024 - OCT_01_2023, // tau
-                0,                         // eta
-                address(0)                 // mgr
-            )
-        );
-        // VoteWizard | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 216.00 MKR | 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
-        VestLike(MCD_VEST_MKR_TREASURY).restrict(
-            VestLike(MCD_VEST_MKR_TREASURY).create(
-                VOTEWIZARD,                // usr
-                216 ether,                 // tot
-                OCT_01_2023,               // bgn
-                SEP_30_2024 - OCT_01_2023, // tau
-                0,                         // eta
-                address(0)                 // mgr
-            )
-        );
+        // Send 2889 DAI from Surplus Buffer to 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c
+        DssExecLib.sendPaymentFromSurplusBuffer(SPARK_AAVE, 2889);
```

Comment:
All DAI values are specified with 2 decimals (as requested recently)

Reply:
The number supplied to `sendPaymentFromSurplusBuffer` [can only be integer](https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssExecLib.sol#L1073-L1082). We can use `ether` keyword hack as with MKR, but only if we abandon `DssExecLib` helper for DAI transfers. Instead, as we've never been instructed to transfer decimal amounts of DAI, I would actually suggest to revert the new rule for DAI payments and only require it for MKR/other tokens

Reply:
cc @amusingaxl

Reply:
Right. We didn't take `DssExecLib` into consideration.
We should probably change that rule for Dai payments made from the Surplus Buffer.

Reply:
We could write it as `2889_00 / 100`, but it's kind of cumbersome. Let's update the checklist.